autoscheduler/frontend/src/components/SchedulingPage/Schedule/ScheduleCard/ScheduleCard.tsx

```diff
@@ -95,8 +98,14 @@ const ScheduleCard: React.FC<ScheduleCardProps> = ({
       onMouseEnter={(): void => setHovered(true)}
       onMouseLeave={(): void => setHovered(false)}
       onMouseDown={(e): void => {
+        // Don't allow dragging in availability blocks
         if (onDragHandleDown && e.target === e.currentTarget) e.stopPropagation();
+        // Ignore mouse down if we have an onClick handler (for MeetingCard -> CourseCard jumping)
+        // which prevents creating an availbility on top of a meeting
+        if (onClick) e.stopPropagation();
       }}
+      onClick={onClick}
+      onKeyPress={onClick}
```

Comment:
Why is this also an `onKeyPress` event? It doesn't seem to do anything and if I'm reading this right, the intention is to go to the section if you press a button over a meeting card (unless I'm reading it wrong)?

Reply:
I did it because otherwise you get `jsx-a11y/click-events-have-key-events`: `Visible, non-interactive elements with click handlers must have at least one keyboard listener ` and just kind of assumed it would work tbh